### ISSUE TITLE
some suggestions for sokol_app from an article

### ISSUE THREAD

**Issue by questor** [floooh/sokol#622]
have you seen this: https://www.anthropicstudios.com/2022/01/13/asking-windows-nicely/ ?
at least the powerschema seems not to be implemented in your wrapper, dpi I've seen you have already integrated. and maybe the "requesting the better gpu" also makes sense at least as an option for windows... (not sure how that is handled in linux)

**Comment by floooh** [comment]
Yes I've read it. For the GPU type there's already a ticket here:

https://github.com/floooh/sokol/issues/485

I'm not sure about the system power setting, at least on Windows, apparently this is a global setting.

On macOS there's a "Quality of Service Class" mechanism where the app can provide hints to the scheduler (especially important on M1 Macs with Performance vs Efficiency cores): https://developer.apple.com/library/archive/documentation/Performance/Conceptual/power_efficiency_guidelines_osx/PrioritizeWorkAtTheTaskLevel.html
